Makefile

```diff
@@ -0,0 +1,19 @@
+# This makefile uses only POSIX 'make' features, except for the
+# following widely implemented extensions -- the .PHONY target, which
+# marks targets that don't generate a file whose name is the same as the
+# target, and prerequisites that depend on glob expansion.
+
+.POSIX:
+.SUFFIXES:
+
+.DEFAULT: preview
+.IGNORE: preview
+.PHONY: preview
+preview:
+	swift package --disable-sandbox preview-documentation --target TSPL
+
+.PHONY: archive
+archive: .build/plugins/Swift-DocC/outputs/TSPL.doccarchive
+
+.build/plugins/Swift-DocC/outputs/TSPL.doccarchive: Sources/TSPL/TSPL.docc/*/*.md
+	xcrun swift package plugin generate-documentation --target TSPL --transform-for-static-hosting
```

Comment:
It would be great to make this work on Linux as well. Also, as of Swift 5.7, `--transform-for-static-hosting` is no longer required, it's passed by default.

Reply:
Changed in 5a8889e2.  I don't have a good way to test this on Linux, but we should definitely work on that.